Code/autopkglib/CodeSignatureVerifier.py

```diff
@@ -174,25 +209,16 @@ def process_app_bundle(self, path):
                 "DISABLE_CODE_SIGNATURE_VERIFICATION to a non-empty value.")
 
         if self.env.get('expected_authority_names', None):
-            self.output("WARNING: Using 'expected_authority_names' to verify .app "
-                        "bundles is deprecated and may be removed in a future "
-                        "AutoPkg release. Verifying .app bundles should use the "
+            self.output("ERROR: Using 'expected_authority_names' to verify code "
+                        "signature is deprecated. Recipes should use the "
```

Comment:
'deprecated' implies 'still works but you should stop using this'. Shouldn't this be "ERROR: Using 'expected_authority_names' to verify code signature is no longer supported..."

Reply:
Yep, updated the wording.

Reply:
So it still works, then? I thought you had removed support. If so, 'deprecated' is not the right word here.

Reply:
I changed "deprecated" to "no longer supported" in cf9367c1e21736d6a5eb60eba73f8f9129471b36. The error is only raised when verifying code signatures (with `codesign`), not installer package signatures (`pkgutil`). I need to write a better description for `expected_authority_names` in input variables section.